demo/data.js

```diff
@@ -6,7 +6,7 @@
  */
```

Comment:
May be worth to add a message body for the commit `[IMP] pivots: introduce calculated measures`